service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProvider.java

```diff
@@ -61,8 +57,50 @@ protected AbstractEmailContentProvider(String uiBaseUrl) {
             .put("fullStop", fullStop)
             .put("timeFramePresent", timeFramePresent)
             .put("timeFrameValue", Optional.ofNullable((String) hearing.get("timeFrame")).orElse(""))
-            .put(DIRECTIONS_KEY, !directions.isEmpty() ? "^" + directions : "")
+            .putAll(directionsArray.build())
             .put("reference", String.valueOf(caseDetails.getId()))
             .put("caseUrl", uiBaseUrl + "/case/" + JURISDICTION + "/" + CASE_TYPE + "/" + caseDetails.getId());
     }
+
+    private boolean buildOrders(Map orders, ImmutableMap.Builder<String, String> orderTypeArray) {
```

Comment:
I agreed with Olivia to stop using callout in notification template and replace it with bullet points.

Please reimplement below logic to prepare one single list of items that will be send.

Long story short content provider will put all orders and directions in one list:

```
List<String> ordersAndDirections = ...

return ImmutableMap.<String, String>builder()
    .put("ordersAndDirections", ordersAndDirections)
...
```

Then in the template we will just have:

```
((ordersAndDirections))
```

It will work like that because Notify enabled support for list recently that are rendered as bullet points.

That change should massively simplify the way orders and directions are handled in code.